api/metrics/community.py

```diff
@@ -35,7 +35,7 @@ def community_areas(self):
             area_number,
             name,
             part
-        FROM community_area
+        FROM community_area\
```

Comment:
@sperveen4 and @Andrew-Tolentino looks like a stray newline got in. I don't think that will cause the query to fail, and the tests pass. So you could remove it when you do your next PR to add the metric to the endpoint and the app.